app/src/main/java/com/android/academy/fundamentals/solutions/SecondActivity_Solution.kt

```diff
@@ -0,0 +1,27 @@
+package com.android.academy.fundamentals.solutions
+
+import android.os.Bundle
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.android.academy.fundamentals.R
+
+class SecondActivity_Solution : AppCompatActivity() {
```

Comment:
Have to rename because it can happen that students will import the wrong activity.